src/molecules/ControlsCompletedCTA.jsx

```diff
@@ -0,0 +1,54 @@
+import { Link, useLocation } from "react-router-dom";
+import { Doughnut } from "react-chartjs-2";
+import { Chart as ChartJS, ArcElement } from "chart.js";
+
+ChartJS.register(ArcElement);
+
+export default function ControlsCompletedCTA({
+  numControlsComplete,
+  numControlsTotal,
+}) {
+  const params = useLocation();
+  const controlsUrl = params.pathname + "/controls";
+  const numControlsIncomplete = numControlsTotal - numControlsComplete;
+  const percentControlsComplete = Math.floor(
+    (numControlsComplete / numControlsTotal) * 100
+  ); // round down to prevent inaccurate "100% complete" feedback as num complete controls near total
+  const data = {
```

Comment:
Nitpik: I know this is exactly how chartjs sets up their examples but a variable name called `data` reminds me of a variable name called `variable` that I saw once.  It feels too generic to me.